modules/renderer/features.js

```diff
@@ -221,15 +221,20 @@ export function rendererFeatures(context) {
         return (geometry === 'line' || geometry === 'area');
     });
 
-
+    defineRule('custom', function isCustom(tags) {
+        const [key, value] = prefs('map-features-custom').split('=');
```

Comment:
I love this feature, I'm using it in a fork of iD. There is a small bug though, `prefs` can return `null` when iD is opened for the first time, which causes a crash here. Something like this would fix it:

```suggestion
        const [key, value] = prefs('map-features-custom')?.split('=') || [];
```